.openshiftio/application.yaml

```diff
@@ -62,16 +59,14 @@ objects:
         type: Source
         sourceStrategy:
           from:
-            kind: ImageStreamTag
-            name: 'runtime:latest'
+            kind: DockerImage
```

Comment:
is this the preferred way to do this?  instead of using the ImageStream?  i'm not an expert on this, and i was just going with what the launcher team had done.

Reply:
Good remark. Certainly not sure if its best practice however insted of of maintaining two image streams (builder image + application image), one could work. We want to control the overall triggers insted of events trigger the changes in deployments.